pact/coin-contract/coin.repl

```diff
@@ -1,217 +1,454 @@
 (begin-tx)
 
-(load "coin-sig.pact")
 (load "coin.pact")
 
 (commit-tx)
 (begin-tx)
 
-(verify 'coin)
+;; HUGE GLARING TODO - pending FV fixes in Pact
+;(verify 'coin)
```

Comment:
TODO here - FV improvements are on the way, verification switched off in the meantime